core/input/input_event.h

```diff
@@ -71,18 +71,20 @@ class InputEvent : public Resource {
 	float get_action_strength(const StringName &p_action, bool p_exact_match = false) const;
 	float get_action_raw_strength(const StringName &p_action, bool p_exact_match = false) const;
 
-	// To be removed someday, since they do not make sense for all events
-	virtual bool is_pressed() const;
+	// To be removed someday, since it doesn't make sense for all events
```

Comment:
This comment was for is_pressed I suppose ? Not sure it still make sense here.

Reply:
The comment was for `is_pressed()` and `is_echo()`, not it's just for `is_echo()`.